lib/email/receiver.rb

```diff
@@ -1359,6 +1359,12 @@ def create_post_with_attachments(options = {})
     def add_attachments(raw, user, options = {})
       raw = raw.dup
 
+      previous_upload_ids =
```

Comment:
We first retrieve the `upload_ids` in the current topic and use that list to ensure we're only adding "new" attachments.